scripts/zones/Mount_Zhayolm/mobs/Sarameya.lua

```diff
@@ -0,0 +1,120 @@
+-----------------------------------
+-- Area: Mount Zhayolm
+-- NPC: Sarameya
+-- @pos 322 -14 -581 61
+-- Spawned with Buffalo Corpse: @additem 2583
+-- Wiki: http://ffxiclopedia.wikia.com/wiki/Sarameya
+-- TODO: PostAIRewrite: Code the Howl effect and gradual resists.
+-----------------------------------
+
+require("scripts/globals/magic");
+require("scripts/globals/status");
+
+-----------------------------------
+-- onMobInitialize Action
+-----------------------------------
+
+function onMobInitialize(mob)
+    mob:setMobMod(MOBMOD_GA_CHANCE, 50);
+    mob:setMobMod(MOBMOD_ADD_EFFECT, mob:getShortID());

```

Comment:
I've gotten it to work with just 1 there. But we've all been using short ID for a long time because thats what the 1st mob add effect scripts did and well, delicious copyapasta.